### DISCUSSION TITLE
How do I toggle draft mode for a PR?

### DISCUSSION THREAD

**Discussion by adegeo** [octokit/octokit.net#2854] [Q&A]
I can't seem to find an class/method/type/whatever that can flip an existing PR into and out of draft mode. Where is this?

**Comment by kfcampbell** [comment]
It looks like we support it when [creating a PR](https://github.com/octokit/octokit.net/blob/9bd519fc42bdef9ddcfce1ea53b242f8b7f2ce0c/Octokit/Models/Request/NewPullRequest.cs#L78), but not when updating. You could submit an issue requesting the feature be added, or do it yourself in a PR if you'd like!
